app/tests/algorithms_tests/test_views.py

```diff
@@ -1367,3 +1370,104 @@ def test_algorithm_model_version_management(settings, client):
     assert not m2.is_desired_version
     del alg.active_model
     assert not alg.active_model
+
+
+@pytest.mark.django_db
+def test_job_list_row_template_ajax_renders(client):
+
+    editor = UserFactory()
+
+    algorithm = AlgorithmFactory()
+    algorithm.add_editor(editor)
+
+    algorithm_image = AlgorithmImageFactory(algorithm=algorithm)
+
+    job = AlgorithmJobFactory(creator=editor, algorithm_image=algorithm_image)
+
+    ctrl_data = workstation_session_control_data(
+        workstation=algorithm.workstation,
+        context_object=algorithm,
+        algorithm_job=job,
+        config=algorithm.workstation_config,
+    )
+
+    headers = {"HTTP_X_REQUESTED_WITH": "XMLHttpRequest"}
+
+    response = get_view_for_user(
+        viewname="algorithms:job-list",
+        client=client,
+        method=client.get,
+        reverse_kwargs={
+            "slug": algorithm.slug,
+        },
+        user=editor,
+        data={
+            "draw": "1",
+            "columns[0][data]": "0",
+            "columns[0][name]": "",
+            "columns[0][searchable]": "false",
+            "columns[0][orderable]": "false",
+            "columns[0][search][value]": "",
+            "columns[0][search][regex]": "false",
+            "columns[1][data]": "1",
+            "columns[1][name]": "",
+            "columns[1][searchable]": "true",
+            "columns[1][orderable]": "true",
+            "columns[1][search][value]": "",
+            "columns[1][search][regex]": "false",
+            "columns[2][data]": "2",
+            "columns[2][name]": "",
+            "columns[2][searchable]": "true",
+            "columns[2][orderable]": "true",
+            "columns[2][search][value]": "",
+            "columns[2][search][regex]": "false",
+            "columns[3][data]": "3",
+            "columns[3][name]": "",
+            "columns[3][searchable]": "true",
+            "columns[3][orderable]": "true",
+            "columns[3][search][value]": "",
+            "columns[3][search][regex]": "false",
+            "columns[4][data]": "4",
+            "columns[4][name]": "",
+            "columns[4][searchable]": "true",
+            "columns[4][orderable]": "true",
+            "columns[4][search][value]": "",
```

Comment:
I don't think you need all this data to run the test, do you? Restrict it to the minimum of what's necessary, or else it wont be clear what you're testing here.

Reply:
Indeed, I updated the tests with minimum parameter needed for the view test